inst/help/summarystatsgeneralbayesiantests.md

```diff
@@ -1,7 +1,7 @@
 Summary Statistics General Bayesian Tests
 ====================================
 
-The general Bayesian tests allow one to test a hypothesis about a parameter under different data likelihoods.
+The general Bayesian tests allow one to test a hypothesis about a parameter under different data likelihoods and are based on the methodology developed and promoted by Zoltan Dienes (e.g., Dienes 2014; Dienes & Mclatchie, 2018).
```

Comment:
"data likelihoods" is by the way odd. A likelihood function is the pdf/pmf fixed at the data treated as a function of the parameters. What about:

The general Bayesian tests allow one to test a hypothesis about a parameter for various distributions and are based on the methodology developed and promoted by Zoltan Dienes (e.g., Dienes 2014; Dienes & Mclatchie, 2018).

Reply:
sure. Any merging issues? (I didn't see them on my end before either)

Reply:
"for various likelihood functions"?

Reply:
updated